package/Linux/gateway/debian/postrm

```diff
@@ -12,6 +12,10 @@ fi
 # System-wide package configuration.
 DEFAULTS_FILE="/etc/default/devolutions-gateway"
 
-if [ -s "$DEFAULTS_FILE" ]; then
-  rm "$DEFAULTS_FILE" || exit 1
+if [ -s $DEFAULTS_FILE ]; then
+  rm $DEFAULTS_FILE || exit 1
+fi
+
+if [ -d /etc/devolutions-gateway ]; then
+  rm -rf /etc/devolutions-gateway
 fi
```

Comment:
To check.
Removing a package using `apt-get remove` should not purge the configuration. Configuration files found in `/etc`, are controlled by the system administrator, and could contain a significant amount of human effort.
However, `apt-get purge` should result in the configuration files to be deleted as well.